### PR TITLE
Brand kit updates

### PR DESCRIPTION
**Pull request by AKCodez**

### CODE REVIEW

**Review by amarantha-k — COMMENTED** [review]
The local build looked good. 
Wondering if the zip files should be under `assets` instead of the `content` folder? @mDuo13 what do you think?

**Comment by AKCodez** [comment]
> The local build looked good. Wondering if the zip files should be under `assets` instead of the `content` folder? @mDuo13 what do you think?

I put it in content as that is where it was being hosted before.

File: content/XRPL_Brand_Kit.zip
Comment (mDuo13) [inline content/XRPL_Brand_Kit.zip:1]: I don't love that this zip has a `__MACOSX/` folder at the top and `.DS_Store` files in every dir, but it's fine I guess.

**Review by mDuo13 — APPROVED** [review]
It's OK for the brand kit to be in `content/` because we're linking people to download it from GitHub via the `github.com/.../raw/...` link. However, if we moved it under `assets/`, then we could serve it directly from xrpl.org at a URL like `https://xrpl.org/assets/XRPL_Brand_Kit.zip`, which _might_ make it easier to track how many times it's been downloaded (using the server access log or something).

File: content/XRPL_Brand_Kit.zip
Comment (amarantha-k) [reply content/XRPL_Brand_Kit.zip:1]: While not a blocker, I'd prefer that we remove these files as they sometimes can cause problems.

**Review by amarantha-k — APPROVED** [review]

File: dactyl-config.yml [hunk 1]
```diff
@@ -4945,6 +4945,21 @@ pages:
...
+        blurb: Review guidelines for using XRPL design assets.
+        top_nav_grouping: Development
+        targets:
+            - en
+
+    -   name: XRPLブランドキット
+        html: https://github.com/XRPLF/xrpl-dev-portal/raw/master/content/XRPL_Brand_Kit.zip
+        parent: resources.html
+        blurb: XRPLのデザインアセットのガイドラインを確認しましょう。
+        top_nav_grouping: Development
```
Comment (tequdev) [inline dactyl-config.yml]:
```suggestion
        top_nav_grouping: 開発
```

In `targets: ja`, "開発" is used for “Development"

https://github.com/XRPLF/xrpl-dev-portal/blob/db87dc69635b18c0d16bb8eab680a3b86258599b/dactyl-config.yml#L4788

**Comment by AKCodez** [comment]
> It's OK for the brand kit to be in `content/` because we're linking people to download it from GitHub via the `github.com/.../raw/...` link. However, if we moved it under `assets/`, then we could serve it directly from xrpl.org at a URL like `https://xrpl.org/assets/XRPL_Brand_Kit.zip`, which _might_ make it easier to track how many times it's been downloaded (using the server access log or something).

Moved into assets

**Pull request merged**